src/database/pipelines/importAiSummaries.ts

```diff
@@ -0,0 +1,85 @@
+import { readFile } from 'fs/promises';
+import { parse } from 'csv-parse';
+import { argv } from 'process';
+import { Kysely, Transaction } from 'kysely';
+import { DB } from '@/database/allDbTypes';
+import { createDB } from '@/database/kyselyDb';
+
+class ImportAiSummaries {
+  public static async run(inputFileName: string, db: Kysely<DB>) {
+    try {
+      await db.transaction().execute(async (trx) => {
+        await new ImportAiSummaries(inputFileName, trx).run();
+      });
+    } catch (error) {
+      console.error(`Error during ImportAiSummaries`, error);
+      throw error;
+    }
+  }
+
+  private constructor(
+    private readonly inputFileName: string,
+    private readonly trx: Transaction<DB>,
+  ) {}
+
+  private async run() {
+    if (!this.inputFileName)
+      throw new Error(`Invalid file name ${this.inputFileName}`);
+
+    const candidateIdLookup = await this.getCandidateIdLookup();
+    const inputStream = await this.createInputStream();
+
+    const summaries = new Array<SummaryRow>();
+    for await (const inputRow of inputStream) {
+      const agendaItemNumber = candidateIdLookup.get(+inputRow.agendaItemId);
```

Comment:
question: why the `+`?

Reply:
yeah this line is cursed.
agendaItemNumber is the human readable ID for the agenda item (which appears in the TMMIS UI and ours and the URLs for the TMMIS API). But the `agendaItemID` is the internal numeric ID.
Since this is coming from a CSV tho, everything is a string.

So this line finds the `agendItemNumber: string` using the `agendaItemId: number` but needs to force that ID into a number from a string

Reply:
ohhh, so is `+` basically acting as a call to `parseInt`/`Number` here? couuuld we use `parseInt` to be more explicit?

Reply:
New CSV includes the reference column. So the cursed line in question is gone